core/templates/ingest.yml

```diff
@@ -129,6 +128,11 @@ paths:
     ingest_sequence_output_render:          '@sequence_publish_area_image/{snapshot_type}/@image_subdirs/@ingest_sequence_output_name.{SEQ}.{img_extension}'
     ingest_shot_output_render:              '@shot_publish_area_image/{snapshot_type}/@image_subdirs/@ingest_shot_output_name.{SEQ}.{img_extension}'
 
+    # PROXIES
+    project_proxy_image:                    '@project_publish_area_image/{snapshot_type}/@proxy_subdirs/@ingest_project_output_name.{SEQ}.{img_extension}'
+    sequence_proxy_image:                   '@sequence_publish_area_image/{snapshot_type}/@proxy_subdirs/@ingest_sequence_output_name.{SEQ}.{img_extension}'
+    shot_proxy_image:                       '@shot_publish_area_image/{snapshot_type}/@proxy_subdirs/@ingest_shot_output_name.{SEQ}.{img_extension}'
+
```

Comment:
Added here, since ingest templates understand some keys (snapshot_type etc.) that will decide the proxy location as well. Template not prefixed with "ingest_" because these are supposed to be common across roles (the same proxies will be created by ingest and read by other roles)

Reply:
I should have caught this earlier, but the ingest output* paths should all be defined as aliases back to the main publish templates. i.e.
```
ingest_project_output_render:        'project_asset_publish_render'
ingest_sequence_output_render:   'sequence_asset_publish_render'
ingest_shot_output_render:            'shot_asset_publish_render'
```

As `snapshot_type` should be able to be mapped to `asset_type`. In which case, we should defined these as:
```
project_asset_publish_proxy
sequence_asset_publish_proxy
shot_asset_publish_proxy
```
In the main template.yml ,which ingest can then either reference directly or create the aliases:
```
ingest_project_output_proxy
ingest_sequence_output_proxy
ingest_shot_output_proxy
```

Reply:
@garvitverma , please clarify about this. Should this be taken to a separate conversation/pull request?